mfr/providers/osf/provider.py

```diff
@@ -84,6 +84,9 @@ def __init__(self, request, url):
         return streams.ResponseStreamReader(response, unsizable=True)
 
     async def _fetch_download_url(self):
+        # Waterbutler url provided
```

Comment:
Minor: can you update the comment to specify that we're looking for a v1 WaterButler url?